teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(age, specialization)
+    super(age)
+    @specialization = specialization
+  end
```

Comment:
- [ ] you will also need to initialize the name and parent permission in this class.

Reply:
like the same, I have done in person?

Reply:
- [ ] You didn't address this comment, Yes the same as you did in person.